app/design/frontend/base/default/template/ebanx/form/fields.phtml

```diff
@@ -40,6 +40,25 @@ if ($helper->hasDocumentFieldAlreadyForMethod($this->getMethodCode()) && (!isset
       </select>
     </div>
     <?php endif ?>
+
+    <?php if ($country === 'CO') : ?>
+    <label for="ebanx-document-type-<?= $this->getMethodCode() ?>" class="ebanx-document-label required">
```

Comment:
This block of code is very similar to the one right on top of it. We should turn it into a single block and fill the select's options dynamically.